src/array/cpu/spmm.cc

```diff
@@ -192,6 +222,56 @@ template void SpMMCsrHetero<kDLCPU, int64_t, 64>(
     const std::vector<dgl_type_t>& ufeat_node_tids,
     const std::vector<dgl_type_t>& out_node_tids);
 
+template void Edge_softmax_csr_forward<kDLCPU, int32_t, 16>(
```

Comment:
Why declaring 16bits here?

Reply:
I have changed the Edge_softmax_csr  declaring  location follow the format like SpMMCsr